distributed/dashboard/components/scheduler.py

```diff
@@ -276,25 +276,30 @@ class MemoryColor:
     orange: float
     red: float
 
-    def __init__(self):
+    def __init__(self, neutral_color="blue", target_color="orange", terminated_color="red"):
+        self.neutral_color = neutral_color
+        self.target_color = target_color
+        self.terminated_color = terminated_color
+
         target = dask.config.get("distributed.worker.memory.target")
         spill = dask.config.get("distributed.worker.memory.spill")
         terminate = dask.config.get("distributed.worker.memory.terminate")
```

Comment:
Assuming these variables don't come up when deciding GPU to CPU spilling and can probably be ignored in that case, not sure what the best way to do this is? First thought was to just use a context manager to override config when initializing the `MemoryColor`, not sure if that will screw up AMM behavior down the line